Source/Bifrost.Client.Specs/Bifrost.Client.Specs.csproj

```diff
@@ -39,30 +39,32 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\Solutions\packages\Castle.Core.3.3.3\lib\net45\Castle.Core.dll</HintPath>
     </Reference>
-    <Reference Include="Machine.Specifications">
-      <HintPath>..\Solutions\packages\Machine.Specifications.0.9.1\lib\net45\Machine.Specifications.dll</HintPath>
+    <Reference Include="Machine.Specifications, Version=0.11.0.0, Culture=neutral, processorArchitecture=MSIL">
```

Comment:
Runner is updated but do not have same version number as the other packages for Machine.Specifications.